wfl/Profiles/RestaurantProfile.cs

```diff
@@ -26,6 +27,23 @@ protected override void Configure(EntityConfiguration<Restaurant> configuration)
                         )
                     )
                )
+               .UseActionTransform(actions => actions
+                    .WithName("edit-restaurant")
+                    .WithRepresentation("restaurant")
+                    .WithMethod(ActionMethod.Replace)
+                    .WithEncoding("application/x-www-form-urlencoded")
+                    .WithLink<Restaurant, RestaurantsController>(r => c => c.UpdateAsync(null, r.Id))
+                    .WithField(x => x
+                        .WithName(nameof(RestaurantDto.Name))
+                        .WithType("text")
+                        .WithTitle("Name")
+                    )
+                    .WithField(x => x
+                        .WithName(nameof(RestaurantDto.MenuLink))
```

Comment:
Being that this is a URI, we can have hypermedia and HTML5 work together to use the proper input field types. Check out https://www.w3schools.com/html/html5_new_elements.asp and a new input type is `url` which browsers can implement URL validation for on the client side.